### PR TITLE
Contrast 13239 disable super admin

### PR DESCRIPTION
**Pull request by kaitlynmortimer**
https://contrast.atlassian.net/browse/CONTRAST-13239?filter=-1

### CODE REVIEW

**Comment by kaitlynmortimer** [comment]
Please correct as needed, particularly titles and other labels.

File: content/admin/teamserver/config/settings/Disable-superadmin-access.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<!--
+title: "Disable SuperAdmin Access for EOP Nodes"
+description: "EOP guide for disabling SuperAdmin accounts"
+tags: "SuperAdmin EOP disable permissions"
+-->
+
+The following steps lead you through the process of disabling SuperAdmin access from Enterprise-on-Premises (EOP) nodes that are accessible to general users.  
+
+##Disable Access for Specific EOP Nodes
```
Comment (terra4383) [inline content/admin/teamserver/config/settings/Disable-superadmin-access.md]: I think there needs to be a space after the ##

File: content/admin/teamserver/config/settings/Disable-superadmin-access.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+To disable SuperAdmin access from generally accessible nodes, begin by running Contrast in a [distributed setup](https://docs.contrastsecurity.com/admin_tsinstall.html#dist) as a new installation. You'll only be prompted to disable SuperAdmin if you're installing as an *application-only installation*; a full installation won't have this option.
+
+If you already have a distributed Contrast setup, add `-Dsuper.admin.disabled=(false or true)` to `$CONTRAST_HOME/bin/contrast-server.vmoptions`. If false, the node allows SuperAdmin logins. If true, the node doesn't allow SuperAdmin logins. 
+
+>**Note:** Contrast supports one secret node.
+
+##Disable Access for SAML
```
Comment (terra4383) [inline content/admin/teamserver/config/settings/Disable-superadmin-access.md]: @thedunster Do we need to touch on configuration in the UI here?

File: content/admin/teamserver/config/settings/Disable-superadmin-access.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+The following steps lead you through the process of disabling SuperAdmin access from Enterprise-on-Premises (EOP) nodes that are accessible to general users.  
+
+##Disable Access for Specific EOP Nodes
+
+To disable SuperAdmin access from generally accessible nodes, begin by running Contrast in a [distributed setup](https://docs.contrastsecurity.com/admin_tsinstall.html#dist) as a new installation. You'll only be prompted to disable SuperAdmin if you're installing as an *application-only installation*; a full installation won't have this option.
+
+If you already have a distributed Contrast setup, add `-Dsuper.admin.disabled=(false or true)` to `$CONTRAST_HOME/bin/contrast-server.vmoptions`. If false, the node allows SuperAdmin logins. If true, the node doesn't allow SuperAdmin logins. 
+
+>**Note:** Contrast supports one secret node.
```
Comment (terra4383) [inline content/admin/teamserver/config/settings/Disable-superadmin-access.md]: @thedunster Does this mean we can only support ONE secret node per installation? If so, @kaitlynmortimer maybe we make this sentence clearer.

Comment (thedunster) [inline content/admin/teamserver/config/settings/Disable-superadmin-access.md]: Yes

File: content/admin/teamserver/config/settings/Disable-superadmin-access.md (hunk 2 above)
Comment (thedunster) [inline content/admin/teamserver/config/settings/Disable-superadmin-access.md]: I'm not sure how deep we'd would want to go. Maybe in the SAML docs we can say if you are trying to configure saml with a secret node instance you will be provided with two sets of metadata? One for the public nodes and one for the secret node. I can send you some screenshots @kaitlynmortimer of that page if you like.

Comment (kaitlynmortimer) [inline content/admin/teamserver/config/settings/Disable-superadmin-access.md]: Will add a note to the SSO article. If you can send a screenshot, that would be lovely.

**Pull request merged**